src/app/views/staff/staff.controller.coffee

```diff
@@ -1,20 +1,35 @@
-@App.controller 'StaffController', ($filter, $stateParams, $uibModal, toastr, MnoeUsers, MnoAppsInstances) ->
+@App.controller 'StaffController', ($log, $stateParams, $window, $uibModal, MnoeUsers, MnoeSubTenants, toastr) ->
   'ngInject'
   vm = this
+  vm.isSaving = false
+  # Get the user
+  MnoeUsers.get($stateParams.staffId).then(
+    (response) ->
+      vm.staff = response.data
+      if(vm.staff.mnoe_sub_tenant_id)
+        MnoeSubTenants.get(vm.staff.mnoe_sub_tenant_id).then((r) -> vm.staff.subTenants = r.data.sub_tenants)
+  )
+  MnoeSubTenants.list(null, null, null).then((response) -> vm.subTenants = response.data)
```

Comment:
I am a bit squeezed here as I actually need all of them for the combo box. What would you suggest ?